src/NServiceBus.Core/EndpointCreator.cs

```diff
@@ -131,7 +133,9 @@ public void UseExternallyManagedBuilder(IBuilder builder)
 
         public async Task<IStartableEndpoint> CreateStartableEndpoint()
         {
-            pipelineComponent.Initialize(containerComponent.Builder);
+            var builder = containerComponent.Builder;
```

Comment:
Why was this change made? Seems like the only reason was to avoid accessing the property multiple times?

Reply:
Yea, that was the reason, do you see any drawbacks? (happy to go back to the old structure)

Reply:
Removing some property accesses is sort of micro-optimization, so I wasn't sure if there was something I was missing.

Reply:
yea, gave it another think and since this happens once at startup there is no need to optimize so I reverted to the old way which gives a better picture when U do a "show all usages" on `ContainerComponent.Builder`